### ISSUE TITLE
How to add an SEO image? 

### ISSUE THREAD

**Issue by abeburnett** [business-science/portfoliodown#4]
Great package, Matt. I'm making great progress creating my portfolio with it. Say, when I share a link to my portfolio on mobile, I'd love to have a slick image pop up. I think this is called an "SEO sharing image" or something like that. I looked at the blogdown documentation and don't see anything particular about how to do this. Do you know if it's possible? 

And, if it's possible, how can I implement this?

Basically the value of being able to do this is that sharing links via iMessage or on social media looks a lot better. Otherwise it's just a header with the link itself.

Thanks in advance!

**Comment by mdancho84** [comment]
I'm pretty sure this is possible.

**Comment by abeburnett** [comment]
Any idea where I'd look (or who I could talk to) to find out how to potentially do this? I'm happy to report back here whatever I find... @mdancho84
